modcc/printer/printerutil.hpp

```diff
@@ -94,9 +92,21 @@ struct module_variables_t {
 
 module_variables_t local_module_variables(const Module&);
 
+// "normal" procedures in a module.
+// A normal procedure is one that has been declared with the
+// PROCEDURE keyword in NMODL.
+
+std::vector<ProcedureExpression*> module_normal_procedures(const Module& m);
+
 // Extract key procedures from module.
 
 APIMethod* find_api_method(const Module& m, const char* which);
 
 NetReceiveExpression* find_net_receive(const Module& m);
 
+struct indexed_variable_info {
+    std::string data_var;
```

Comment:
Why the var postfix? would data and index suffice ?

Reply:
An "indexed variable" is really two variables when printing: the array (the data variable) and the index (the index variable).
This struct doesn't talk about the data and index, it describes the data variable and index variable that are going to be used in the printed output/
Does that make sense?